src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -399,7 +399,10 @@ protected override bool TryGetCompilerDiagnosticCode(string diagnosticId, out ui
 
         private protected override Compilation RunGenerators(Compilation input, ParseOptions parseOptions, ImmutableArray<AdditionalText> additionalTexts)
         {
-            //PROTOTYPE: this needs to be behind a feature flag
+            if (((CSharpParseOptions)parseOptions).LanguageVersion != LanguageVersion.Preview)
```

Comment:
We usually add a feature name and then register that name for a particular language version. Why should this check be different?

Reply:
Jared said no point, as we'll drop it before release. 

---
In reply to: [387320422](https://github.com/dotnet/roslyn/pull/42030#discussion_r387320422) [](ancestors = 387320422)